src/dataset-builder/ConceptSelector.ts

```diff
@@ -41,7 +41,7 @@ export const findRoot = <T extends RowContents>(parents: Parent<T>[]) => {
 };
 
 export const ConceptSelector = (props: ConceptSelectorProps) => {
-  const { title, onCancel, onCommit, actionText, snapshotId, initialCart, parents, openedConcept } = props;
+  const { title, onCancel, onCommit, snapshotId, initialCart, parents, openedConcept } = props;
```

Comment:
Why are you able to remove `actionText` from here? I would expect that it would be unbound on line 105 below

Reply:
Honestly just a mistake. I was poking at it, because action test was often the same except for in tests. But decided that wasn't worth the changes to all the tests.